tests/Unit/PointwiseFunctions/GeneralRelativity/Test_ComputeGhQuantities.cpp

```diff
@@ -574,4 +579,131 @@ SPECTRE_TEST_CASE("Unit.PointwiseFunctions.GeneralRelativity.GhQuantities",
                                     upper_bound);
   test_shift_deriv_functions_analytic(solution, grid_size, lower_bound,
                                       upper_bound);
+
+  // Check that compute items work correctly in the DataBox
+  // First, check that the names are correct
+  CHECK(GeneralizedHarmonic::Tags::TimeDerivSpatialMetricCompute<
+            3, Frame::Inertial>::name() == "dt(SpatialMetric)");
+  CHECK(GeneralizedHarmonic::Tags::TimeDerivLapseCompute<
+            3, Frame::Inertial>::name() == "dt(Lapse)");
+  CHECK(GeneralizedHarmonic::Tags::TimeDerivShiftCompute<
+            3, Frame::Inertial>::name() == "dt(Shift)");
+
+  // Check that the compute items return the correct values
+  const DataVector test_vector{5., 4.};
+  auto spatial_metric =
+      make_with_value<tnsr::ii<DataVector, 3, Frame::Inertial>>(test_vector,
+                                                                0.);
+  get<0, 0>(spatial_metric) = 1.5;
+  get<0, 1>(spatial_metric) = 0.1;
+  get<0, 2>(spatial_metric) = 0.2;
+  get<1, 1>(spatial_metric) = 1.4;
+  get<1, 2>(spatial_metric) = 0.2;
+  get<2, 2>(spatial_metric) = 1.3;
+
+  auto lapse = make_with_value<Scalar<DataVector>>(test_vector, 0.94);
+
+  auto shift =
+      make_with_value<tnsr::I<DataVector, 3, Frame::Inertial>>(test_vector, 0.);
+  get<0>(shift) = 0.5;
+  get<1>(shift) = 0.6;
+  get<2>(shift) = 0.7;
+
+  auto dt_spatial_metric =
+      make_with_value<tnsr::ii<DataVector, 3, Frame::Inertial>>(test_vector,
+                                                                0.);
+  get<0, 0>(dt_spatial_metric) = 0.05;
+  get<0, 1>(dt_spatial_metric) = 0.01;
+  get<0, 2>(dt_spatial_metric) = 0.02;
+  get<1, 1>(dt_spatial_metric) = 0.04;
+  get<1, 2>(dt_spatial_metric) = 0.02;
+  get<2, 2>(dt_spatial_metric) = 0.03;
+
+  auto dt_lapse = make_with_value<Scalar<DataVector>>(test_vector, 0.);
+  get(dt_lapse) = 0.04;
+
+  auto dt_shift =
```

Comment:
The values which are only used to compare against the compute item can be given variable names like `expected_dt_shift` for clarity.

I would also prefer the tests be similar to #1478, where the computed functions tested elsewhere are trusted, and this just be verifying the compute item infrastructure. That way, random values can also be used, which would improve robustness (avoid repeated values in the components), and be easier to maintain.

Reply:
I have changed the compute item tests to be similar to #1478 - i.e. using random values. Also, renamed relevant variables by prepending `expected_` to them.

Reply:
I squashed this @moxcodes , given how similar the changes were to #1478 and #1477 :)